hoot-services/src/main/java/hoot/services/controllers/grail/GrailResource.java

```diff
@@ -218,7 +218,7 @@ private InternalCommand getPublicOverpassCommand(String jobId, File workDir, Gra
      *
      * {
      *   //The upper left (UL) and lower right (LR) of the bounding box to clip the dataset
-     *   "BBOX" : "{"LR":[-77.04813267598544,38.89292259454q727],"UL":[-77.04315011486628,38.89958152667718]}",
+     *   "bounds" : "{"LR":[-77.04813267598544,38.89292259454q727],"UL":[-77.04315011486628,38.89958152667718]}",
```

Comment:
Can you add as a comment somewhere in the code examples of what hoot-services accepts for the poly param vs the bbox param?
and then if either of those formats differ from what is sent to overpass and what is sent to hoot convert.bounds?
i.e. does hoot convert.bounds accept the same poly param value as overpass?
I don't think this current format is actually used anywhere?

Reply:
overpass does not but we handle that logic so user doesn't need to know that.

I will add the correct format in comments